utils/tokenutils.go

```diff
@@ -0,0 +1,95 @@
+package utils
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"errors"
+	"sync"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+)
+
+type Claims struct {
```

Comment:
Curious why you need this struct since it just wraps jwt.StandardClaims. I don't see any extra methods defined on Claims.

Reply:
You're right. Previously didn't realize that the `NotBefore` time was included in `StandardClaims`. Removed.